Mathlib/CategoryTheory/Monoidal/Functorial.lean

```diff
@@ -47,36 +47,69 @@ open MonoidalCategory
 variable {C : Type u₁} [Category.{v₁} C] [MonoidalCategory.{v₁} C] {D : Type u₂} [Category.{v₂} D]
   [MonoidalCategory.{v₂} D]
 
--- Perhaps in the future we'll redefine `LaxMonoidalFunctor` in terms of this,
--- but that isn't the immediate plan.
-/-- An unbundled description of lax monoidal functors. -/
-class LaxMonoidal (F : C → D) [Functorial.{v₁, v₂} F] where
+/-- An unbundled description of lax monoidal functors without axioms. See `LaxMonoidal` for
+the full description. -/
+class LaxMonoidalStruct (F : C → D) [Functorial.{v₁, v₂} F] where
   /-- unit morphism -/
   ε : 𝟙_ D ⟶ F (𝟙_ C)
   /-- tensorator -/
   μ : ∀ X Y : C, F X ⊗ F Y ⟶ F (X ⊗ Y)
-  /-- naturality -/
-  μ_natural :
-    ∀ {X Y X' Y' : C} (f : X ⟶ Y) (g : X' ⟶ Y'),
-      (map F f ⊗ map F g) ≫ μ Y Y' = μ X X' ≫ map F (f ⊗ g) := by
-   aesop_cat
+
+-- Perhaps in the future we'll redefine `LaxMonoidalFunctor` in terms of this,
+-- but that isn't the immediate plan.
+/-- An unbundled description of lax monoidal functors. -/
+class LaxMonoidal (F : C → D) [Functorial.{v₁, v₂} F] extends LaxMonoidalStruct F where
+  μ_natural_left :
+    ∀ {X Y : C} (f : X ⟶ Y) (X' : C),
+      (map F f ▷ F X') ≫ μ Y X' = μ X X' ≫ map F (f ▷ X') := by
+    aesop_cat
+  μ_natural_right :
+    ∀ {X Y : C} (X' : C) (f : X ⟶ Y) ,
+      (F X' ◁ map F f) ≫ μ X' Y = μ X' X ≫ map F (X' ◁ f) := by
+    aesop_cat
   /-- associativity of the tensorator -/
   associativity :
     ∀ X Y Z : C,
-      (μ X Y ⊗ 𝟙 (F Z)) ≫ μ (X ⊗ Y) Z ≫ map F (α_ X Y Z).hom =
-        (α_ (F X) (F Y) (F Z)).hom ≫ (𝟙 (F X) ⊗ μ Y Z) ≫ μ X (Y ⊗ Z) := by
-    aesop_cat
-  /-- left unitality -/
-  left_unitality : ∀ X : C, (λ_ (F X)).hom = (ε ⊗ 𝟙 (F X)) ≫ μ (𝟙_ C) X ≫ map F (λ_ X).hom := by
+      (μ X Y ▷ F Z) ≫ μ (X ⊗ Y) Z ≫ map F (α_ X Y Z).hom =
+        (α_ (F X) (F Y) (F Z)).hom ≫ (F X ◁ μ Y Z) ≫ μ X (Y ⊗ Z) := by
     aesop_cat
-  /-- right unitality -/
-  right_unitality : ∀ X : C, (ρ_ (F X)).hom = (𝟙 (F X) ⊗ ε) ≫ μ X (𝟙_ C) ≫ map F (ρ_ X).hom := by
-    aesop_cat
-#align category_theory.lax_monoidal CategoryTheory.LaxMonoidal
-
-attribute [simp] LaxMonoidal.μ_natural
+  left_unitality : ∀ X : C, (λ_ (F X)).hom = (ε ▷ F X) ≫ μ (𝟙_ C) X ≫ map F (λ_ X).hom :=
+    by aesop_cat
+  right_unitality : ∀ X : C, (ρ_ (F X)).hom = (F X ◁ ε) ≫ μ X (𝟙_ C) ≫ map F (ρ_ X).hom :=
+    by aesop_cat
 
-attribute [simp] LaxMonoidal.μ_natural
+-- Perhaps in the future we'll redefine `LaxMonoidalFunctor` in terms of this,
+-- but that isn't the immediate plan.
+open LaxMonoidalStruct in
+/-- An unbundled description of lax monoidal functors. -/
+def LaxMonoidal.ofTensorHom (F : C → D) [Functorial.{v₁, v₂} F] [LaxMonoidalStruct F]
```

Comment:
```suggestion
abbrev LaxMonoidal.ofTensorHom (F : C → D) [Functorial.{v₁, v₂} F] [LaxMonoidalStruct F]
```
since otherwise this isn't reducibly defeq to its argument.